oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -87,6 +88,8 @@ if __name__ == '__main__':
 Please install CMAPI first.', file=sys.stderr)
                 sys.exit(1)
             try:
+                # Delay to allow SM to start up
+                time.sleep(10)
```

Comment:
this sleep happens w/ or w/o SM. Perhaps the timeout on line 68, which is executed after starting SM should be increased and we can remove this timeout and timeout on line 73

Reply:
We can even move it to line 58, removing this one and sleep from line 73.